autoload/peekaboo.vim

```diff
@@ -75,7 +75,7 @@ function! s:append_group(title, regs)
         continue
       endif
       let s:regs[printf('%s', r)] = line('$')
-      call append(line('$'), printf(' %s: %s', r, val))
+      call append(line('$'), printf(' %s: %s', r, substitute(val, '^\s*\(.\{-}\)\s*$', '\1', '')))
```

Comment:
No need to trim trailing spaces, so we can simplify this to:

```vim
substitute(val, '^\s*', '', '')
```

Reply:
I added one more commit to trim only leading spaces as you described.